### ISSUE TITLE
Playlist info command fails

### ISSUE THREAD

**Issue by lol768** [np1/mps-youtube#275]
To reproduce:

* Search for any playlist ("`pls test`")
* Request info on a playlist with "`i 1`"

**TypeError**: Can't convert 'NoneType' object to str implicitly

```
Traceback (most recent call last):
  File "./mpsyt", line 3, in <module>
	mps_youtube.main.main()
  File "/tmp/mps_youtube/main.py", line 4735, in main
    if matchfunction(k, v, userinput):`
  File "/tmp/mps_youtube/main.py", line 4636, in matchfunction
    func(*matches)
  File "/tmp/mps_youtube/main.py", line 4137, in info
    out += ("\n\nAuthor     : " + p['author'])
```

**Comment by ids1024** [comment]
Fixed in 2eef024bc7730e678670d0e7359e4e8b104eff4.

**Comment by lol768** [comment]
Confirmed fixed, thanks.
